src/layouts/sidebarRight.vue

```diff
@@ -5,14 +5,14 @@
                 Header Should come here.
             </slot>
         </div>
-        <div class="right-panel-wrapper" :class="{ 'hidden': rightHidden }">
+      <!--  <div class="right-panel-wrapper" :class="{ 'hidden': rightHidden }">
```

Comment:
Uncomment this, we are anyways not going to use this file for admin.

Reply:
ok